lib/classic/index.js

```diff
@@ -0,0 +1,43 @@
+const awilix = require('awilix')
+const fp = require('fastify-plugin')
+
+const diContainer = awilix.createContainer({
```

Comment:
Could it give issue if the plugin is registered twice on different contexts?

```
fastify.register(async (instance)=>{
  instance.register(this plugin)
})
fastify.register(async (instance)=>{
  instance.register(this plugin)
})
```

Reply:
Fair point. I will expose an option to pass user's own container for such cases.